bases/strictest.json

```diff
@@ -14,7 +14,6 @@
 
     "isolatedModules": true,
 
-    "checkJs": true,
```

Comment:
Why not just set allowJs explicitly false here? Then, downstream users can turn it on as though the default were false.

Reply:
TS doesn't allow that

```sh
 Option 'checkJs' cannot be specified without specifying option 'allowJs'.
 ```